kernel/kernel-config-4.9.x

```diff
@@ -603,7 +603,7 @@ CONFIG_PCI_MSI_IRQ_DOMAIN=y
 # CONFIG_HT_IRQ is not set
 CONFIG_PCI_ATS=y
 # CONFIG_PCI_IOV is not set
-# CONFIG_PCI_PRI is not set
+CONFIG_PCI_PRI=y
```

Comment:
Do we know we need this? I looked it up - it allows PCI devices behind an iommu to recover from page faults - sounds a little 'fringe' for our virtio case?

Reply:
@grahamwhaley Yes you are right, we dont typically need this for the virtio case, taking this out.